src/manifolds/CenteredMatrices.jl

```diff
@@ -119,7 +119,7 @@ where $c_i = \frac{1}{m}\sum_{j=1}^m x_{j,i}$  for $i = 1, \dots, n$.
 """
 project(::CenteredMatrices, ::Any, ::Any)
 
-project!(M::CenteredMatrices, Y, p, X) = (Y .= X .- mean(X, dims=1))
+project!(::CenteredMatrices, Y, p::Any, X) = (Y .= X .- mean(X, dims=1))
```

Comment:
Do we need `::Any` here?

Reply:
If we leave out the `p` since the variable is not used, then `::Any` would be nice, sure.

Reply:
I think we generally tend to leave `::Any` and keep variable names for points and tangent vectors, to make the code more descriptive.

Reply:
ah right, we only did any for doctoring signatures I think. I will switch it to just `p`